src/QMCTools/CMakeLists.txt

```diff
@@ -69,6 +68,23 @@ TARGET_LINK_LIBRARIES(convertpw4qmc PRIVATE qmcutil Math::FFTW3)
 INSTALL(TARGETS convertpw4qmc
         RUNTIME DESTINATION bin)
 
+SET(FSSRCS
+  QMCFiniteSize/QMCFiniteSize.cpp
+  QMCFiniteSize/SkParserBase.cpp
+  QMCFiniteSize/SkParserASCII.cpp
+  QMCFiniteSize/SkParserScalarDat.cpp
+  QMCFiniteSize/FSUtilities.cpp
+)
+
+ADD_LIBRARY(fstool ${FSSRCS})
+TARGET_LINK_LIBRARIES(fstool PUBLIC qmc qmcbase qmcwfs)
```

Comment:
at the moment, it only needs einspline. this will change in the future when a kinetic energy FS correction is added, which will depend on the 2body jastrow and a kspace jastrow. if it is preferable to just add the various .cpp files in FSSRCS that each thing depends on I can do that instead as opposed to targetting libraries that include those dependencies

Reply:
Let us put einspline and change it to qmcwfs when actual WFS dependence comes in.

Reply:
Thanks. I see. The dependency is needed right now.